### ISSUE TITLE
Most listings are broken

### ISSUE THREAD

**Issue by linqcan** [kodi-svtplay/xbmc-svtplay#186]
```
[TestSvt] __get_json : 'ERROR: Failed to get JSON for http://svtplay.se/api/all_titles'
F[TestSvt] __get_json : 'ERROR: Failed to get JSON for http://svtplay.se/api/all_titles'
E........[TestSvt] __get_json : 'ERROR: Failed to get JSON for http://svtplay.se/api/all_titles'
F.[TestSvt] __get_json : 'ERROR: Failed to get JSON for http://svtplay.se/api/all_titles'
F.
```

**Comment by linqcan** [comment]
First broken build: https://travis-ci.org/linqcan/xbmc-svtplay/builds/195253635

**Comment by linqcan** [comment]
Fixed in 4.0.41, pull requested.
